x/mongo/driver/batch_cursor.go

```diff
@@ -308,23 +308,36 @@ func (bc *BatchCursor) KillCursor(ctx context.Context) error {
 	}.Execute(ctx)
 }
 
+// calcGetMoreBatchSize returns the number of documents that should be returned in
+// the response of a "getMore" operation given the limit, batchSize, and number
+// of documents already returned.
+func calcGetMoreBatchSize(ctx context.Context, bc BatchCursor) (int32, bool) {
```

Comment:
The context parameter is unused.

```suggestion
func calcGetMoreBatchSize(bc BatchCursor) (int32, bool) {
```

Reply:
I didn't quite get the motivation of changing this function from a method of `BatchCursor` to an independent function.

Reply:
The purpose is to unit test the calculation.